examples/3d/solari.rs

```diff
@@ -61,9 +64,38 @@ fn setup(
     >,
 ) {
     commands
-        .spawn(SceneRoot(asset_server.load(
-            GltfAssetLabel::Scene(0).from_asset("models/CornellBox/CornellBox.glb"),
-        )))
+        .spawn((
+            SceneRoot(
+                asset_server.load(
+                    GltfAssetLabel::Scene(0)
+                        .from_asset("https://github.com/bevyengine/bevy_asset_files/raw/2a5950295a8b6d9d051d59c0df69e87abcda58c3/pica_pica/mini_diorama_01.glb")
```

Comment:
Maybe add a command line flag to the example (something like `--allow-web-assets`) before downloading stuff, letting users verify the assets and helping them not accidentally download things if they're on, say, a very limited connection.

Reply:
Ehhh, they already have to compile with the https feature. I don't really want to get into making the example fancier, I just want a better demo for 0.17 than the cornell box...

Reply:
Ah ok, that's fine then.